src/variableSubstitution.ts

```diff
@@ -99,9 +99,9 @@ const VARIABLE_SUBSTITUTIONS: VariableSubstitution[] = [
 export function getVariableSubstitutedAssociations(associations: XMLFileAssociation[]): XMLFileAssociation[] {
 
   // Collect properties needed to resolve variables
-  const currentFile: TextDocument = window.activeTextEditor.document;
-  const currentFileUri: string = currentFile && currentFile.uri.fsPath;
-  const currentWorkspace: WorkspaceFolder = workspace.getWorkspaceFolder(currentFile && currentFile.uri);
+  const currentFile: TextDocument = window.activeTextEditor && window.activeTextEditor.document && window.activeTextEditor.document.languageId === 'xml' && window.activeTextEditor.document;
```

Comment:
Sorry this code doesn't work,it currentFile is equals to false when the condition is not respected although it should return undefined.